tracekit.d.ts

```diff
@@ -87,13 +91,13 @@ interface Report {
      * Add a crash handler.
      * @param {Function} handler
      */
-    subscribe(handler:(stackTrace:StackTrace, options?:any) => void): void;
+    subscribe(handler: ReportSubscriber): void;
```

Comment:
I believe `options`, though declares as `any`, was a `boolean`. It's the `inWindowError` second argument of a handler.